server/routes/lecture_booking-route.js

```diff
@@ -0,0 +1,13 @@
+const express = require("express");
+
+// Import lecture-controller
+const lectureRoutes = require("./../controllers/lecture_booking-controller");
+
+// Create router
+const router = express.Router();
+
+router.get("/studentbooking", lectureRoutes.getBookingLectures);
```

Comment:
The naming used makes it hard to understand the purpose of the end-point.
I would move this under `lecture-router` and change the naming in the following way:
```js
router.get("/bookable/:userId", ...
```